plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityAnger.java

```diff
@@ -3,115 +3,81 @@
 import com.denizenscript.denizen.objects.EntityTag;
 import com.denizenscript.denizencore.objects.core.DurationTag;
 import com.denizenscript.denizencore.objects.Mechanism;
-import com.denizenscript.denizencore.objects.ObjectTag;
-import com.denizenscript.denizencore.objects.properties.Property;
-import com.denizenscript.denizencore.objects.properties.PropertyParser;
 import org.bukkit.entity.Bee;
-import org.bukkit.entity.Entity;
 import org.bukkit.entity.PigZombie;
 
-public class EntityAnger implements Property {
+public class EntityAnger extends EntityProperty<DurationTag> {
 
-    public static boolean describes(ObjectTag entity) {
-        if (!(entity instanceof EntityTag)) {
-            return false;
-        }
-        Entity bukkitEntity = ((EntityTag) entity).getBukkitEntity();
-        return bukkitEntity instanceof PigZombie
-                || bukkitEntity instanceof Bee;
-    }
+    // <--[property]
+    // @object EntityTag
+    // @name anger
+    // @input DurationTag
+    // @description
+    // Controls the remaining anger time of a PigZombie or Bee.
+    // -->
 
-    public static EntityAnger getFrom(ObjectTag entity) {
-        if (!describes(entity)) {
-            return null;
-        }
-        else {
-            return new EntityAnger((EntityTag) entity);
-        }
+    public static boolean describes(EntityTag entity) {
+        return entity.getBukkitEntity() instanceof Bee
+                || entity.getBukkitEntity() instanceof PigZombie;
     }
 
-    public static final String[] handledMechs = new String[] {
-            "anger"
-    };
+    EntityTag entity;

```

Comment:
My bad if I'm missing something, but was this tested? it looks like the old field was kept but isn't ever set, I.e. the entity is always `null`.
Generally the old fields, `is/getPigZombie` sorta methods, and stuff like `getAnger` can just be removed in favor of doing it directly with stuff like `instanceof` pattern matching, like:
```java
if (entity instanceof ZombiePig zombie) {
    zombie.doThing()
}
else {
    as(Bee).doThing();
}
```
But should at least remove the old field and make the existing methods use `EntityProperty#getEntity`.